### PR TITLE
Spanish novia713

### PR DESCRIPTION
**Pull request by novia713**
faltaban ficheros correspondientes a algunos comandos

### CODE REVIEW

**Comment by jmolivas** [comment]
I tagged a new version 0.11.3 and sync commands thats why this PR has some conflicts.

**Comment by novia713** [comment]
merged from _0.11.3_ & applied _generate:doc:gitbook_

File: es/about/documentación-para-traductores.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+
```
Comment (jmolivas) [inline es/about/documentación-para-traductores.md]: Should we used an English name and add a blank page or a translated version on `en/about/` directory

**Comment by novia713** [comment]
hecho!

**Comment by novia713** [comment]
merged from master & ready for success ^^

**Pull request merged**